### PR TITLE
Added Skeleton: Classifier, Preprocessing, Interface classes

### PR DESCRIPTION
**Pull request by SkanderSoltani**
In sentiment_classifier.py
  - Added Skeleton + simple classifier for testing pipeline
  - Added Preprocessing class 
  - Added Interface class to test: text_to_sentiment()

- playground.py is to check how training work including saving checkpoints
- Added config.yaml file

### CODE REVIEW

File: climate_scanner/sentiment/playground.py [hunk 1]
```diff
@@ -0,0 +1,40 @@
+from sentiment_classifier import *
+
+def get_text():
+    path = get_data('extracted_text.parquet.gzip')
+    text = pd.read_parquet(path).dropna()
```
Comment (Eliotfrazier) [inline climate_scanner/sentiment/playground.py:5]: Probably fine for now, but pandas dataframes can be pretty slow when processing large volumes of data.

Comment (Eliotfrazier) [reply climate_scanner/sentiment/playground.py:5]: A dictionary might be a better choice here.

Comment (Eliotfrazier) [reply climate_scanner/sentiment/playground.py:5]: Assuming this is a test script anyway :)

File: climate_scanner/sentiment/sentiment_classifier.py [hunk 2]
```diff
@@ -0,0 +1,387 @@
...
+        # Method defining the forward pass in the classifier model (called internally during forward pass in compile method)
+        # Args:
+        #   inputs     ->  np.array() representing tokens 
+        # returns:
+        #   lastTensor -> tensor representing last layer in the model    
+        
+        x = self.Embedding(inputs)
+        x = self.LSTM(x)
+        x = self.dense1(x)
+        lastTensor = self.dense2(x)
```
Comment (Eliotfrazier) [inline climate_scanner/sentiment/sentiment_classifier.py:145]: Softmax?

File: climate_scanner/sentiment/sentiment_classifier.py [hunk 3]
```diff
@@ -0,0 +1,387 @@
...
+        #    call_back_dir   -> optional, of type str describing name of log directory for callbacks, default is None
+        # Returns:
+        #    ---
+        if call_backs_dir:
+            # tensorboard -log files 
+            log_dir = get_data(self._path_to_logs)+call_backs_dir + '\\'+datetime.now().strftime("%Y%m%d-%H%M%S")
+            tensorboard_callback = keras.callbacks.TensorBoard(log_dir=log_dir)
+            
+            # checkpoints to save model
+            checkpoint_dir = get_data(self._path_to_checkpoints) +call_backs_dir +'\\'
```
Comment (Eliotfrazier) [inline climate_scanner/sentiment/sentiment_classifier.py:168]: Us os.path.join(), rather than + here, e.g. 

```
log_dir = get_data(os.path.join(self._path_to_logs, call_backs_dir , datetime.now().strftime("%Y%m%d-%H%M%S")))
```

**Review by Eliotfrazier — APPROVED** [review]
Looking pretty good to me so far!
